src/server/render.ts

```diff
@@ -209,6 +233,17 @@ export async function render<Data>(
   // Clear previous slots
   SLOTS_TRACKER.clear();
 
+  // Clear rendering state
+  RENDERING_USER_TEMPLATE = false;
```

Comment:
RENDERING_USER_TEMPLATE should not be defined as screaming snake case.

Reply:
It's in uppercase to signal that it's a global and not a local variable.